pkg/systemd/init.js

```diff
@@ -1269,16 +1269,18 @@ $(function() {
         var unit = "[Unit]\nDescription=";
         var service = "\n[Service]\nExecStart=";
         var timer = "\n[Timer]\n";
-        var service_file = unit + timer_unit.Description + service + timer_unit.Command + "\n[Install]\nWantedBy=default.target";
+        var install = "[Install]\nWantedBy=default.target\n";
```

Comment:
`timers.target` would be the canonical location, or `multi-user.target` if it shouldn't be started for "degenerate" modes like single-user/rescue. But I think `timers.target` would be most appropriate.